ui/plugin-system/src/runtime/time-series-queries.ts

```diff
@@ -140,3 +144,33 @@ function useTimeSeriesQueryContext(): TimeSeriesQueryContext {
     refreshKey,
   };
 }
+
+/**
+ * Get active time series queries for query results summary
+ */
+export function useActiveTimeSeriesQueries() {
+  const queryClient = useQueryClient();
+  const queryCache = queryClient.getQueryCache();
+  const timeSeriesQueries = getActiveTimeSeriesQueries(queryCache);
+  return timeSeriesQueries;
+}
+
+/**
+ * Filter all cached queries down to only active time series queries
+ */
+export function getActiveTimeSeriesQueries(cache: QueryCache) {
```

Comment:
Would it be beneficial to add a unit test for this bit of logic?

Reply:
This one is kind of difficult to write a unit test for, I tried mocking what is returned in queryCache, but it's probably better to use msw so started on that, but it's becoming a bigger effort since we haven't used msw yet in Perses. 

I'll probably come back to this later when we go to make the validation / warnings banner more generic--at that point, I'll make an e2e test as well.